tests/unit/specs/components/ActionModal/components/__snapshots__/ActionModal.spec.js.snap

```diff
@@ -29,6 +29,8 @@ exports[`ActionModal should show the action modal on success 1`] = `
     <!---->
   </div>
    
+  <!---->
```

Comment:
should we have one test showing the warning?

Reply:
I'm writing it now. Having a little trouble getting the snapshot to output something useful.